.github/contributing.md

```diff
@@ -0,0 +1,24 @@
+# How to properly make a request in Minehut Meta
+
+So, you want to suggest a feature, report a bug, or maybe suggest a change for the filter? Always remember to follow these guidelines: 
+
+1. Use the issue templates provided
```

Comment:
this is required by forms no? we probably don't need this here

Reply:
You mean the entire file? I only have it there because it puts a popup saying `Hey! Read the guidelines for issues` or smth similar